en/flight_controller/modalai_fc_v1.md

```diff
@@ -100,6 +100,21 @@ To [build PX4](https://dev.px4.io/master/en/setup/building_px4.html) for this ta
 make modalai_fc-v1
 ```
 
+
+## Serial Port Mapping
+
```

Comment:
This is what was available from the DavidS recipe applied to ModalAI. Don't know UART4 or UART8

Reply:
@modaltb Please fill this in

Reply:
@hamishwillee how can I push update?  

UART | Device | Port
--- | --- | ---
USART1 | /dev/ttyS0 | GPS1 (J10)
USART2 | /dev/ttyS1 | TELEM3 (J4)
USART3 | /dev/ttyS2 | Debug Console (J2)
UART4 | /dev/ttyS3 | Expansion UART (J6)
UART5 | /dev/ttyS4 | TELEM2, Primary VOXL Communications (J1)
USART6 | /dev/ttyS5 | RC (J12)
UART7 | /dev/ttyS6 | TELEM1 (J5)
UART8 | /dev/ttyS7 | N/A

Reply:
Thanks very much @modaltb. I've pushed it for you.

For next time, you **should** be able to just edit the file directly within the PR and push back to the branch. 
![image](https://user-images.githubusercontent.com/5368500/77020534-374f0780-69d8-11ea-9527-7eb3e42e791e.png)